content/docs/04.5-devx/07-sandbox-clusters.md

```diff
@@ -47,3 +47,57 @@ To create your new sandbox cluster:
    * Review the information and deploy the Palette virtual cluster. The Palette virtual cluster will be provisioned within the next few minutes.
 
 <br />
+
+
+## Palette Virtual Cluster Pause and Resume
+
+The Palette allows the Pause and Resume of Palette Virtual Clusters when not in use. This feature enables the users with optimized resource sharing by pausing the virtual clusters, not in use. This adds significant flexibility in managing operating costs and resource management for the Palette virtual clusters. 
+
+<br />
+
+### Resource Dependencies
+
+* The Quota allocation is independent of a Virtual Cluster's Pause or Resume status.
```

Comment:
I'm not sure I understand what this means. When I read the sentence, I ask myself, "does this mean that the user's quota is always in effect?" Meaning that regardless of the state of a cluster (paused/active), the requested CPU and memory are still counted towards the overall user's quota limit?

Additionally,  quota and virtual clusters should not be in the title case.